proto/viam/app/v1/app.proto

```diff
@@ -975,3 +979,23 @@ message CreateKeyResponse {
   string key = 1;
   string id = 2;
 }
+
+message DeleteKeyRequest {
+  string key = 1;
+  string id = 2;
```

Comment:
is it typical to request both the id and key to delete an API key? this looks good to me but I just wanted to double check to see that this is the usual pattern

Reply:
Yeah I think this is related to the comments on Roxy's related PRs and is explained in the spec for org api keys (under "Why do we need both a key and an ID field for the ID? Isn't a single cryptographically generated key already safe?"). Let me know if you still have questions!